### PR TITLE
LRSD 4766 [Learn] Add page banner and  info tags, side nav menu, bottom nav and sign-in cta

### PR DESCRIPTION
**Pull request by luisacmn**

### CODE REVIEW

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/sign-in/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+const isUserSignedIn = themeDisplay.isSignedIn();
```
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/sign-in/index.js:6]: Inline

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/sign-in/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+if (!isUserSignedIn) {
+	const signInContainer = document.querySelector('.signin-container');
+	const signInLink = document.createElement('a');
+	const signInMessage = document.createElement('span');
+
+	signInLink.textContent = 'Sign In';
+	signInLink.setAttribute(
+		'href',
+		'/home?p_p_id=com_liferay_login_web_portlet_LoginPortlet&p_p_lifecycle=0&p_p_state=maximized&p_p_mode=view&_com_liferay_login_web_portlet_LoginPortlet_mvcRenderCommandName=%2Flogin%2Flogin&saveLastPath=false'
```
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/sign-in/index.js:16]: Use '/c/portal/login'

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/sign-in/index.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+const isUserSignedIn = themeDisplay.isSignedIn();
+
+if (!isUserSignedIn) {
+	const signInContainer = document.querySelector('.signin-container');
```
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/sign-in/index.js:9]: As used

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layout-page-templates/display-page-templates/learn-course-introduction/page-definition.json [hunk 4]
```diff
@@ -32,80 +32,352 @@
...
+															"value": {
+																"fragmentImage": {
+																	"description": {
+																		"value_i18n": {
+																			"en_US": ""
+																		}
+																	},
+																	"url": {
+																		"value_i18n": {
+																			"en_US": "/documents/d/liferay-learn/education_courses_blue-svg?imagePreview=1"
```
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layout-page-templates/display-page-templates/learn-course-introduction/page-definition.json:101]: Use placeholder instead absolute path

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/documents/group/vectors/Waves.svg
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/documents/group/vectors/Waves.svg:1]: Rename to lowercase

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/documents/group/vectors/smallwaves.svg
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/documents/group/vectors/smallwaves.svg:1]: Follow the pattern and rename to waves_sm.svg

**Review by joseabelenda — CHANGES_REQUESTED** [review]

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layout-page-templates/display-page-templates/learn-course-introduction/page-definition.json (hunk 4 above)
Comment (luisacmn) [reply workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layout-page-templates/display-page-templates/learn-course-introduction/page-definition.json:101]: will be done in the icons/images refactoring ticket

**Pull request closed without merging**